docs/modules/plugin/nav.adoc

```diff
@@ -0,0 +1,11 @@
+* Maven Plugin
```

Comment:
I recommend using the terminology "Main Plugin" here, or perhaps "Processor Plugin". Having "Maven Plugin" twice in the breadcrumbs feels confusing.

Reply:
With the "Maven Tools" title renaming this gets solved quite nicely :smile: